### DISCUSSION TITLE
Improve welcome page

### DISCUSSION THREAD

**Discussion by civier** [neurodesk/neurodesk.github.io#122] [Ideas]
I don't think that using the term "browser-based" in the welcome page is wise, as people would assume they cannot use the non-web tools they are used to + assume that it is only a cloud service.
I would change to "A flexible and scalable data analysis virtual desktop for reproducible neuroimaging that comes with all software preinstalled"

Virtual desktop - so it will be clear that this is a graphical environment
With all software preinstalled - this is the main selling point for early users
I would use something more concise for "all software preinstalled" but I couldn't find something clear enough. Any ideas?
The fact that the client is browser-based is nice, but I wouldn't put it at the top of the list.

Also, I think it will be best to include a screenshot of the environment that is easily accessible from the main page (at least, until a demo is available)

**Comment by stebo85** [comment]
adding browser-based come up as a suggestion during brainhack. 

How about:
"A flexible, scalable and browser-accessible neuroimaging analysis environment for reproducible neuroimaging"
